react-events/my-app/src/Button.tsx

```diff
@@ -0,0 +1,7 @@
+type Props = {
+  label: string;
+  onClick: () => void;
+};
+export function Button({ label, onClick }: Props) {
```

Comment:
let's call the onClick prop something else. It can be confused for the onClick event. 
You can use onButtonClick or something similar.